db/migrate/20210718074605_update_ibm_vendor_to_ibm_cloud.rb

```diff
@@ -0,0 +1,25 @@
+class UpdateIbmVendorToIbmCloud < ActiveRecord::Migration[6.0]
+  class VmOrTemplate < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+    self.inheritance_column = :_type_disabled
+    self.table_name = 'vms'
+  end
+
+  def up
+    say_with_time('Updating vendor to "IBM Cloud" for VPC managers') do
+      VmOrTemplate.in_my_region.where(:type => 'ManageIQ::Providers::IbmCloud::VPC::CloudManager::Vm').update_all(:vendor => 'ibm_cloud')
```

Comment:
Think @Fryguy mentioned this but you'll need to update templates as well:
```
VmOrTemplate.in_my_region.where(:type => ['ManageIQ::Providers::IbmCloud::VPC::CloudManager::Vm', 'ManageIQ::Providers::IbmCloud::VPC::CloudManager::Template']).update_all(:vendor => 'ibm_cloud')
```

Reply:
I _think_ I've got it right now. I also added `::Template` checks to the specs.